gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/src/AuthServer.js

```diff
@@ -64,35 +64,77 @@ gpii.oauth2.oauth2orizeServer.listenOauth2orize = function (oauth2orizeServer, c
 
     /**
      * Processes OAuth2 resource owner GPII key grant requests.
-     * @param {Object} clientInfo - The structure is {
-     *     client: {Object},  // all fields from "client" document for this client
-     *     clientCredentialId: {String}  // The internal "id" value from "clientCredential" document that associates with
-     *                                   // the client credential used in this authorization request.
+     * @param {Object} clientInfo - The client and its privilege info.
+     * The structure is {
```

Comment:
Better to use a @typedef for this so that the description of the structure can be linted